src/code_80057C60.c

```diff
@@ -2594,21 +2594,17 @@ void func_8005CB60(s32 playerId, s32 lapCount) {
             }
             *huhthedeuce += 1;
             if (1) {}
-            switch (*huhthedeuce) { /* switch 1; irregular */
-                case 0:             /* switch 1 */
-                    break;
-                case 1:                                   /* switch 1 */
-                    CM_ActivateSecondLapLakitu(playerId); // func_80079084(playerId);
-                    func_800C9060(playerId, SOUND_ARG_LOAD(0x19, 0x00, 0xF0, 0x15));
-                    if ((IsLuigiRaceway()) && (D_80165898 == 0) && (gModeSelection != (s32) TIME_TRIALS)) {
-                        D_80165898 = 1;
-                    }
-                    break;
-                case 2:                                  /* switch 1 */
-                    CM_ActivateFinalLapLakitu(playerId); // func_800790B4(playerId);
-                    break;
-                case 3: /* switch 1 */
-                    if ((D_8018D114 == 0) || (D_8018D114 == 1)) {
+
+            if (*huhthedeuce == 1) {
```

Comment:
maybe just redo the animation of lakitu for each lap to have an animation even if it's incorect

Reply:
Right now, nothing should happen between the extra laps. This could be done. I'm not really sure what's best though.
Like do we really want lakitu to say "2nd" lap when it's wrong? Or "Final lap?" when it's wrong?

Reply:
Perhaps a 'next lap' function could be made. And it gets used if a next lap lakitu texture exists?